ConsumerApi/appsettings.override.json

```diff
@@ -79,6 +79,11 @@
             }
         },
         "Relationships": {
+            "BlobStorage": {
+                "CloudProvider": "Azure",
+                "ConnectionInfo": "",
+                "ContainerName": ""
+            },
```

Comment:
Remove this. Shouldn't be needed, because we removed blob storage for relationships